publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -406,6 +414,9 @@ public void checkAndCreateExchangeIfNeeded() throws RemRemPublishException {
      */
     private boolean hasExchange() throws RemRemPublishException {
         log.info("Exchange is: " + exchangeName);
```

Comment:
Shouldn't this line in fact be place now inside the if(hasExchange), when you really know you have an exchange?